src/app/article-data.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { ArticleFormat } from './format';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ArticleDataService {
+  articleData : ArticleFormat = null;
```

Comment:
`ArticleFormat` can be renamed to `Article`

Reply:
Renamed as suggested